docs/content/docs/guide/statuses.md

```diff
@@ -137,3 +137,17 @@ generating artifacts on every push. The relevant section of the pipeline can be
 found here:
 
 <https://github.com/openshift-pipelines/pipelines-as-code/blob/7b41cc3f769af40a84b7ead41c6f037637e95070/.tekton/push.yaml#L116>
+
+## Tekton Results Annotation
+
+The specialized annotation for Tekton Results is stored in a specific format:
```

Comment:
This can be improved, this does not give use info for why we are adding

Reply:
Hi Piyush 👋🏼, PaC code doesn't use this annotation in any way. This annotation is just for capturing additional information about PR's origins for Tekton Results (cli or UI). This is the information I aimed to put in the doc as well.
Please help me refine it if it's not up to the expectation/clear.

Reply:
This will be used by results, but put some details like, it is getting added by default in PAC. this will store ----thse things if results in enabled etc etc.

Reply:
thinking a bit more about it we can just remove the documentation altogether here, we don't document the internal implementation and end users don't care as much..

wdyt @ramessesii2 ?

Reply:
That sounds right to me. I'm gonna push the suggested changes.